.github/workflows/quarkus.yml

```diff
@@ -0,0 +1,321 @@
+name: Quarkus tests
+
+on: [push]
+
+env:
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  DB_USER: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_NAME: hibernate_orm_test
+  NATIVE_TEST_MAVEN_OPTS: "-B -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
+  MX_GIT_CACHE: refcache
+  JAVA_HOME: ${{ github.workspace }}/openjdk
+
+jobs:
+  build-mandrel-and-quarkus:
+    name: Mandrel and Quarkus build
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+        path: mandrel
+    - uses: actions/checkout@v1
+      with:
+        repository: graalvm/mx.git
+        ref: master
+        path: mx
+    - name: Get latest quarkus release
+      run: |
+        curl --output quarkus.tgz -sL $(curl -sL https://api.github.com/repos/quarkusio/quarkus/releases/latest | jq -r .tarball_url)
```

Comment:
Hmmm, I wonder if we should test against master instead. Master tested with master might be more useful.

Reply:
I agree, I would actually suggest testing both the latest release *and* master.